src/styles/modal.css

```diff
@@ -0,0 +1,83 @@
+/* modal */
+
+.ui.modal {
```

Comment:
I believe we need to add some `padding-bottom` for the smaller screens:

<img width="386" alt="image" src="https://github.com/abby-moffitt/GA-Semantic/assets/101662434/af20cb72-df09-4151-b365-6880d3349bab">

I believe the breakpoint is 768px.

Reply:
That's because this breakpoint has `!important` rule which can''t be overiden easily:
<img width="250" alt="Screenshot 2023-06-13 at 3 52 12 PM" src="https://github.com/abby-moffitt/GA-Semantic/assets/64768811/82a09ee3-cd66-4f17-a49f-323f9ce0ab05">

Reply:
But I found a way around it by using padding